tests/php/UsedOnTableTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SilverStripe\Admin\Tests;
+
+use SilverStripe\Admin\Forms\UsedOnTable;
+use SilverStripe\Assets\File;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Dev\TestOnly;
+
+class UsedOnTableTest extends SapphireTest implements TestOnly
```

Comment:
You don't need to implement TestOnly in test classes (SapphireTest already does), only in stubs used in tests